### PR TITLE
Fix how do we differ section

### PR DESCRIPTION
**Pull request by cp-sumi-k**

### CODE REVIEW

File: vue-frontend/src/components/home/DoYouKnow.vue [hunk 1]
```diff
@@ -92,13 +91,32 @@ export default {
...
+  .que-image {
+    width: 24%;
+  }
+
+  .do-you-know-text {
+    margin: 0 0 15% 14%;
+  }
+
+  .do-you-know-text {
+    padding-left: 0%;
```
Comment (nidhi-canopas) [inline vue-frontend/src/components/home/DoYouKnow.vue:109]: Let's remove duplicate selectors and combine styles into one.

File: vue-frontend/src/components/home/HowDoWeDiffer.vue [hunk 2]
```diff
@@ -302,5 +296,24 @@ export default {
...
+    width: 60%;
+    min-width: 900px;
+  }
+
+  .differ-description {
+    padding: 0 10%;
+  }
+
+  .description {
+    padding-top: 116px;
```
Comment (nidhi-canopas) [inline vue-frontend/src/components/home/HowDoWeDiffer.vue:316]: Should we use static paddings?

File: vue-frontend/src/components/home/DoYouKnow.vue [hunk 3]
```diff
@@ -33,26 +35,28 @@ export default {
...
 
 .flex-row {
   display: flex;
   align-items: center;
 }
 
-.flex-row > .image {
+.que-image {
   display: none;
+  width: 25%;
```
Comment (nidhi-canopas) [inline vue-frontend/src/components/home/DoYouKnow.vue:50]: Do we need width while setting display none?

File: vue-frontend/src/components/home/HowDoWeDiffer.vue (hunk 2 above)
Comment (cp-shruti-s) [reply vue-frontend/src/components/home/HowDoWeDiffer.vue:316]: Because override the section in above components so that used static padding

Comment (nidhi-canopas) [reply vue-frontend/src/components/home/HowDoWeDiffer.vue:316]: Can't we apply in percentage?

**Review by nidhi-canopas — APPROVED** [review]

**Pull request merged**